Makefile.am

```
@@ -150,6 +150,7 @@ scratch:
 	if [ "$(TEST_BUILD)" == "true" ]; then \
 		sed -ri '/AC_INIT/ s/\[[0-9.]+\]/[999999999]/' $(srcdir)/configure.ac; \
 	fi
+	rm -rf ui/webui/dist
```

You should make it ignore the error, probably like so
```suggestion
	-rm -rf ui/webui/dist
```

---

The -f will makes it just work, even when the dir does not exist.

---

From the `rm` man pages:

```
       -f, --force
              ignore nonexistent files and arguments, never prompt
```

---

Thank you, that makes sense.